LICENSE

```diff
@@ -0,0 +1,54 @@
+Copyright 2014 Paulus Schoutsen
```

Comment:
@balloob I'd like to get your permission for this. My understanding is that the copyright date/name should match the date/name of the original contributions.

Reply:
Yeah that's fine.